packages/channels/src/telegram/channel.ts

```diff
@@ -0,0 +1,23 @@
+import { ChannelTemplate } from '../base/channel'
+import { TelegramApi } from './api'
+import { TelegramConfig, TelegramConfigSchema } from './config'
+import { TelegramService } from './service'
+import { TelegramStream } from './stream'
+
+export class TelegramChannel extends ChannelTemplate<TelegramConfig, TelegramService, TelegramApi, TelegramStream> {
+  get meta() {
+    return {
+      // TODO: change the meta here
+      id: '0198f4f5-6100-4549-92e5-da6cc31b4ad1',
+      name: 'telegram',
+      schema: TelegramConfigSchema,
+      initiable: true,
+      lazy: true
+    }
+  }
```

Comment:
I left it like the same metadata of the old telegram channel for now, so you can switch between the old and new implementation. Needs to be changed before merging this